BuildingThreatModel.md

```diff
@@ -0,0 +1,73 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project 2020. -->
+# Building a Threat Model
+
+Threat modeling is a process by which potential threats, such as structural vulnerabilities or the absence of appropriate safeguards, can be identified, enumerated, and mitigation can be prioritized. 
+It is recommended that consumers embrace threat modeling technique and walk through series of concrete steps in order to fully understand the environment and specific solution you are trying to build and secure. 
+
+Thread modeling should at the end answer questions like:
+
+- Where am I most vulnerable to attack? 
+- What are the most relevant threats?
+- What do I need to do to safeguard against these threats?
+
+Egeria's architecture allows consumers to make specific choices and minimize risk factors from the early solution design phase.
```

Comment:
Maybe before this sentence we could say something like: 'Using open metadata in your IT architecture means that you can build on the thinking outlined here by the community around how Egeria deals with different types of threats.